### PR TITLE
Closes #43

### LINKED ISSUE #43: Upgrade to spark 1.2.0
I tried to upgrade to spark 1.2.0 release but somehow checksum fails for 1.2.0 deps. Ideally this may be a lein issue or my local envrionents fault. Anyways flambo's spark deps needs to be changes for upgrade.

### PR DESCRIPTION
**Pull request by kul**
Upgraded spark to 1.2.0

### CODE REVIEW

**Comment by kul** [comment]
@sorenmacbeth The changes are breaking mainly due the rename of BernoulliCellSampler. If PR looks good may be would need a tag.

File: test/flambo/api_test.clj [hunk 1]
```diff
@@ -362,3 +366,9 @@
        (-> (f/parallelize c [1.0 2.2 2.6 3.3 3.5 3.7 4.4 4.8 5.5 6.0])
            (f/histogram [1.0 4.0 6.0])) => [6 4])
       )))
+
```
Comment (sorenmacbeth) [inline test/flambo/api_test.clj]: What is the purpose of this?

File: test/flambo/api_test.clj [hunk 2]
```diff
@@ -145,12 +145,16 @@
...
+        using same seed should return same result."
         (-> (f/parallelize c [0 1 2 3 4 5 6 7 8 9])
-            (f/sample false 0.1 2)
+            (f/sample false 0.3 9)
             f/collect
-            vec) => [6])
+            vec) => (-> (f/parallelize c [0 1 2 3 4 5 6 7 8 9])
+                        (f/sample false 0.3 9)
+                        f/collect
+                        vec))
```
Comment (sorenmacbeth) [inline test/flambo/api_test.clj]: Nice, I like this fix for the test.

File: test/flambo/api_test.clj (hunk 1 above)
Comment (kul) [inline test/flambo/api_test.clj]: Ops I was debugging from repl. I will remove this and squash in a new commit.

**Comment by kul** [comment]
Removed the comment form and squashed the change.

**Comment by sorenmacbeth** [comment]
LGTM

**Pull request merged**